### ISSUE TITLE
".pbm" and "Foreground/Background segmentation:"

### ISSUE THREAD

**Issue by chho-work** [debidatta/syndata-generation#12]
Thanks for sharing the code base!

In the paper, section:
5. Approach Details and Analysis -> Foreground/Background segmentation:
"We train a FCN network [28] (based on VGG-16 [44] pre-trained on PASCAL VOC [8] image segmentation) to classify each image pixel into foreground/background...."

Does it mean that to obtain the mask files like "NP1_123.pbm", we can:
1) Train it on a "FCN network...", just like you mentioned in the paper.
2) Use "Pixel Objectness with Bilinear Filtering" or "Grabcut"(mentioned in the prerequisite of the repo)?  

Sorry, I am a newbie and just couldn't find out how is the FCN network related to this repo.  

Thanking you in advance!

**Comment by debidatta** [comment]
For the paper, we had access to Ground Truth masks in the BigBIRD dataset. So we were able to train an FCN as a foreground/background segmentation model.

However in general these masks might not be available. For the repo, we wanted to give some pointers as to how one can get masks for their dataset. So that is why we mentioned GrabCut/Pixel Objectness with Bilinear Filtering, which do not require training an additional segmentation model. 

Hope this clears the confusion,

**Comment by chho-work** [comment]
Thanks for your prompt reply!

You recommended using Pixel Objectness which  is based in Caffe framework, if I am not mistaken.  As I mentioned, I am new to machine learning and started out learning Pytorch.  To learn another framework right now seems a bit challenging for me.  

Questions:
1) I would like to know if the DSS-Pytorch that you mentioned in another thread will work as Pixel Objectness.  
2) If not, by chance, do you know if Pixel Objectness can be installed in Google Colab, as I have no access to physical GPU nor I subscribe to cloud GPU services.

Many thanks for your support!

**Comment by debidatta** [comment]
You can stick to one framework and use DSS-Pytorch. If your class of interest exists in [PASCAL dataset](http://host.robots.ox.ac.uk/pascal/VOC/voc2012/) you can use https://pytorch.org/hub/pytorch_vision_deeplabv3_resnet101/ . 

If your images have simple background with object in center you can use https://docs.opencv.org/master/d8/d83/tutorial_py_grabcut.html and use the center pixel as the seed.

**Comment by chho-work** [comment]
I appreciate your prompt and complete reply!  
Have a nice day!
